Auto update failing for libgrpc

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [libgrpc](https://github.com/termux/termux-packages/blob/8f54627bb7fd853ad7c29a0df0c7ffc535800c98/packages/libgrpc) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating libgrpc [Current version: 1.73.1]
INFO: package being updated to 1.74.0.
INFO: Trying to build package.
INFO: Package libgrpc exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building libgrpc for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0  81635      0 --:--:-- --:--:-- --:--:-- 82087
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  27895      0 --:--:-- --:--:-- --:--:-- 27924
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  33846      0 --:--:-- --:--:-- --:--:-- 33901
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   2294      0 --:--:-- --:--:-- --:--:--  2301
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Wed 23 Jul 2025 06:16:01 PM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   2166      0 --:--:-- --:--:-- --:--:--  2163
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Sun 20 Jul 2025 06:22:07 AM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   2218      0 --:--:-- --:--:-- --:--:--  2215
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Wed 23 Jul 2025 12:18:06 PM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
::endgroup::
Downloading dependency libc++@27c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libc++/libc++_27c_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  291k  100  291k    0     0  2129k      0 --:--:-- --:--:-- --:--:-- 2127k
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency abseil-cpp@20250127.1 if necessary...
Found abseil-cpp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/a/abseil-cpp/abseil-cpp_20250127.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1082k  100 1082k    0     0  12.5M      0 --:--:-- --:--:-- --:--:-- 12.5M
extracting abseil-cpp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency resolv-conf@1.3 if necessary...
Found resolv-conf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/resolv-conf/resolv-conf_1.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   976  100   976    0     0  17648      0 --:--:-- --:--:-- --:--:-- 17745
extracting resolv-conf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency c-ares@1.34.5 if necessary...
Found c-ares in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/c-ares/c-ares_1.34.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  187k  100  187k    0     0  3078k      0 --:--:-- --:--:-- --:--:-- 3125k
extracting c-ares to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2025.07.15 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates/ca-certificates_1:2025.07.15_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  119k  100  119k    0     0  2256k      0 --:--:-- --:--:-- --:--:-- 2263k
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2025.07.15 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates-java/ca-certificates-java_1:2025.07.15_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  109k  100  109k    0     0  1781k      0 --:--:-- --:--:-- --:--:-- 1801k
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency gflags@2.2.2-9 if necessary...
Found gflags in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gflags/gflags_2.2.2-9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 73560  100 73560    0     0   182k      0 --:--:-- --:--:-- --:--:--  182k
extracting gflags to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gflags-static@2.2.2-9 if necessary...
Found gflags-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gflags-static/gflags-static_2.2.2-9_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 54300  100 54300    0     0   154k      0 --:--:-- --:--:-- --:--:--  154k
extracting gflags-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/z/zlib/zlib_1.3.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 60516  100 60516    0     0  1057k      0 --:--:-- --:--:-- --:--:-- 1074k
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libprotobuf@2:30.0-3 if necessary...
Found libprotobuf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libp/libprotobuf/libprotobuf_2:30.0-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 1255k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1255k  100 1255k    0     0  3443k      0 --:--:-- --:--:-- --:--:-- 3440k
extracting libprotobuf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency protobuf-dev@2:30.0-3 if necessary...
Found protobuf-dev in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/protobuf-dev/protobuf-dev_2:30.0-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  3160  100  3160    0     0  63645      0 --:--:-- --:--:-- --:--:-- 64489
extracting protobuf-dev to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency protobuf@2:30.0-3 if necessary...
Found protobuf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/protobuf/protobuf_2:30.0-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1556k  100 1556k    0     0  19.7M      0 --:--:-- --:--:-- --:--:-- 19.7M
extracting protobuf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libre2@2025-07-22-0 if necessary...
Found libre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libr/libre2/libre2_2025-07-22-0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  162k  100  162k    0     0  2544k      0 --:--:-- --:--:-- --:--:-- 2582k
extracting libre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.5.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl/openssl_1:3.5.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2390k  100 2390k    0     0  26.9M      0 --:--:-- --:--:-- --:--:-- 27.1M
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.5.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl-tool/openssl-tool_1:3.5.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  266k  100  266k    0     0  4504k      0 --:--:-- --:--:-- --:--:-- 4508k
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading libgrpc@1.74.0 git source from 'git+https://github.com/grpc/grpc' if necessary...
Downloading git source with branch 'v1.74.0' from 'https://github.com/grpc/grpc'
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout'...
Note: switching to '3e7a4d52d257990fa5b9d80f69f4a591178d9d7c'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

~/.termux-build/libgrpc/cache/tmp-checkout ~/.termux-build/libgrpc/cache
Submodule 'third_party/abseil-cpp' (https://github.com/abseil/abseil-cpp.git) registered for path 'third_party/abseil-cpp'
Submodule 'third_party/benchmark' (https://github.com/google/benchmark) registered for path 'third_party/benchmark'
Submodule 'third_party/bloaty' (https://github.com/google/bloaty.git) registered for path 'third_party/bloaty'
Submodule 'third_party/boringssl-with-bazel' (https://github.com/google/boringssl.git) registered for path 'third_party/boringssl-with-bazel'
Submodule 'third_party/cares/cares' (https://github.com/c-ares/c-ares.git) registered for path 'third_party/cares/cares'
Submodule 'third_party/envoy-api' (https://github.com/envoyproxy/data-plane-api.git) registered for path 'third_party/envoy-api'
Submodule 'third_party/googleapis' (https://github.com/googleapis/googleapis.git) registered for path 'third_party/googleapis'
Submodule 'third_party/googletest' (https://github.com/google/googletest.git) registered for path 'third_party/googletest'
Submodule 'third_party/opencensus-proto' (https://github.com/census-instrumentation/opencensus-proto.git) registered for path 'third_party/opencensus-proto'
Submodule 'third_party/opentelemetry' (https://github.com/open-telemetry/opentelemetry-proto.git) registered for path 'third_party/opentelemetry'
Submodule 'third_party/opentelemetry-cpp' (https://github.com/open-telemetry/opentelemetry-cpp) registered for path 'third_party/opentelemetry-cpp'
Submodule 'third_party/protobuf' (https://github.com/protocolbuffers/protobuf.git) registered for path 'third_party/protobuf'
Submodule 'third_party/protoc-gen-validate' (https://github.com/envoyproxy/protoc-gen-validate.git) registered for path 'third_party/protoc-gen-validate'
Submodule 'third_party/re2' (https://github.com/google/re2.git) registered for path 'third_party/re2'
Submodule 'third_party/xds' (https://github.com/cncf/xds.git) registered for path 'third_party/xds'
Submodule 'third_party/zlib' (https://github.com/madler/zlib) registered for path 'third_party/zlib'
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/abseil-cpp'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/benchmark'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/boringssl-with-bazel'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/cares/cares'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/envoy-api'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/googleapis'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/googletest'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opencensus-proto'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/protobuf'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/protoc-gen-validate'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/re2'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/xds'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/zlib'...
From https://github.com/abseil/abseil-cpp
 * branch            76bb24329e8bf5f39704eb10d21b9a80befa7c81 -> FETCH_HEAD
Submodule path 'third_party/abseil-cpp': checked out '76bb24329e8bf5f39704eb10d21b9a80befa7c81'
From https://github.com/google/benchmark
 * branch            12235e24652fc7f809373e7c11a5f73c5763fc4c -> FETCH_HEAD
Submodule path 'third_party/benchmark': checked out '12235e24652fc7f809373e7c11a5f73c5763fc4c'
From https://github.com/google/bloaty
 * branch            60209eb1ccc34d5deefb002d1b7f37545204f7f2 -> FETCH_HEAD
Submodule path 'third_party/bloaty': checked out '60209eb1ccc34d5deefb002d1b7f37545204f7f2'
Submodule 'third_party/abseil-cpp' (https://github.com/abseil/abseil-cpp.git) registered for path 'third_party/bloaty/third_party/abseil-cpp'
Submodule 'third_party/capstone' (https://github.com/aquynh/capstone.git) registered for path 'third_party/bloaty/third_party/capstone'
Submodule 'third_party/demumble' (https://github.com/nico/demumble.git) registered for path 'third_party/bloaty/third_party/demumble'
Submodule 'third_party/googletest' (https://github.com/google/googletest.git) registered for path 'third_party/bloaty/third_party/googletest'
Submodule 'third_party/protobuf' (https://github.com/protocolbuffers/protobuf.git) registered for path 'third_party/bloaty/third_party/protobuf'
Submodule 'third_party/re2' (https://github.com/google/re2) registered for path 'third_party/bloaty/third_party/re2'
Submodule 'third_party/zlib' (https://github.com/madler/zlib) registered for path 'third_party/bloaty/third_party/zlib'
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/abseil-cpp'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/capstone'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/demumble'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/googletest'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/protobuf'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/re2'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/zlib'...
From https://github.com/abseil/abseil-cpp
 * branch            5dd240724366295970c613ed23d0092bcf392f18 -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/abseil-cpp': checked out '5dd240724366295970c613ed23d0092bcf392f18'
From https://github.com/aquynh/capstone
 * branch            852f46a467cb37559a1f3a18bd45d5ca8c6fc5e7 -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/capstone': checked out '852f46a467cb37559a1f3a18bd45d5ca8c6fc5e7'
From https://github.com/nico/demumble
 * branch            01098eab821b33bd31b9778aea38565cd796aa85 -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/demumble': checked out '01098eab821b33bd31b9778aea38565cd796aa85'
From https://github.com/google/googletest
 * branch            565f1b848215b77c3732bca345fe76a0431d8b34 -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/googletest': checked out '565f1b848215b77c3732bca345fe76a0431d8b34'
From https://github.com/protocolbuffers/protobuf
 * branch            bc1773c42c9c3c522145a3119e989e0dff2a8d54 -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/protobuf': checked out 'bc1773c42c9c3c522145a3119e989e0dff2a8d54'
Submodule 'third_party/benchmark' (https://github.com/google/benchmark.git) registered for path 'third_party/bloaty/third_party/protobuf/third_party/benchmark'
Submodule 'third_party/googletest' (https://github.com/google/googletest.git) registered for path 'third_party/bloaty/third_party/protobuf/third_party/googletest'
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/protobuf/third_party/benchmark'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/bloaty/third_party/protobuf/third_party/googletest'...
From https://github.com/google/benchmark
 * branch            5b7683f49e1e9223cf9927b24f6fd3d6bd82e3f8 -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/protobuf/third_party/benchmark': checked out '5b7683f49e1e9223cf9927b24f6fd3d6bd82e3f8'
From https://github.com/google/googletest
 * branch            5ec7f0c4a113e2f18ac2c6cc7df51ad6afc24081 -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/protobuf/third_party/googletest': checked out '5ec7f0c4a113e2f18ac2c6cc7df51ad6afc24081'
From https://github.com/google/re2
 * branch            5bd613749fd530b576b890283bfb6bc6ea6246cb -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/re2': checked out '5bd613749fd530b576b890283bfb6bc6ea6246cb'
From https://github.com/madler/zlib
 * branch            cacf7f1d4e3d44d871b605da3b647f07d718623f -> FETCH_HEAD
Submodule path 'third_party/bloaty/third_party/zlib': checked out 'cacf7f1d4e3d44d871b605da3b647f07d718623f'
From https://github.com/google/boringssl
 * branch            c63fadbde60a2224c22189d14c4001bbd2a3a629 -> FETCH_HEAD
Submodule path 'third_party/boringssl-with-bazel': checked out 'c63fadbde60a2224c22189d14c4001bbd2a3a629'
From https://github.com/c-ares/c-ares
 * branch            6360e96b5cf8e5980c887ce58ef727e53d77243a -> FETCH_HEAD
Submodule path 'third_party/cares/cares': checked out '6360e96b5cf8e5980c887ce58ef727e53d77243a'
From https://github.com/envoyproxy/data-plane-api
 * branch            4de3c74cf21a9958c1cf26d8993c55c6e0d28b49 -> FETCH_HEAD
Submodule path 'third_party/envoy-api': checked out '4de3c74cf21a9958c1cf26d8993c55c6e0d28b49'
From https://github.com/googleapis/googleapis
 * branch            fe8ba054ad4f7eca946c2d14a63c3f07c0b586a0 -> FETCH_HEAD
Submodule path 'third_party/googleapis': checked out 'fe8ba054ad4f7eca946c2d14a63c3f07c0b586a0'
From https://github.com/google/googletest
 * branch            52eb8108c5bdec04579160ae17225d66034bd723 -> FETCH_HEAD
Submodule path 'third_party/googletest': checked out '52eb8108c5bdec04579160ae17225d66034bd723'
From https://github.com/census-instrumentation/opencensus-proto
 * branch            4aa53e15cbf1a47bc9087e6cfdca214c1eea4e89 -> FETCH_HEAD
Submodule path 'third_party/opencensus-proto': checked out '4aa53e15cbf1a47bc9087e6cfdca214c1eea4e89'
From https://github.com/open-telemetry/opentelemetry-proto
 * branch            60fa8754d890b5c55949a8c68dcfd7ab5c2395df -> FETCH_HEAD
Submodule path 'third_party/opentelemetry': checked out '60fa8754d890b5c55949a8c68dcfd7ab5c2395df'
From https://github.com/open-telemetry/opentelemetry-cpp
 * branch            ced79860f8c8a091a2eabfee6d47783f828a9b59 -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp': checked out 'ced79860f8c8a091a2eabfee6d47783f828a9b59'
Submodule 'third_party/benchmark' (https://github.com/google/benchmark) registered for path 'third_party/opentelemetry-cpp/third_party/benchmark'
Submodule 'third_party/googletest' (https://github.com/google/googletest) registered for path 'third_party/opentelemetry-cpp/third_party/googletest'
Submodule 'third_party/ms-gsl' (https://github.com/microsoft/GSL) registered for path 'third_party/opentelemetry-cpp/third_party/ms-gsl'
Submodule 'third_party/nlohmann-json' (https://github.com/nlohmann/json) registered for path 'third_party/opentelemetry-cpp/third_party/nlohmann-json'
Submodule 'third_party/opentelemetry-proto' (https://github.com/open-telemetry/opentelemetry-proto) registered for path 'third_party/opentelemetry-cpp/third_party/opentelemetry-proto'
Submodule 'third_party/opentracing-cpp' (https://github.com/opentracing/opentracing-cpp.git) registered for path 'third_party/opentelemetry-cpp/third_party/opentracing-cpp'
Submodule 'third_party/prometheus-cpp' (https://github.com/jupp0r/prometheus-cpp) registered for path 'third_party/opentelemetry-cpp/third_party/prometheus-cpp'
Submodule 'tools/vcpkg' (https://github.com/Microsoft/vcpkg) registered for path 'third_party/opentelemetry-cpp/tools/vcpkg'
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/benchmark'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/googletest'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/ms-gsl'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/nlohmann-json'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/opentelemetry-proto'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/opentracing-cpp'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/prometheus-cpp'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/tools/vcpkg'...
From https://github.com/google/benchmark
 * branch            344117638c8ff7e239044fd0fa7085839fc03021 -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/benchmark': checked out '344117638c8ff7e239044fd0fa7085839fc03021'
From https://github.com/google/googletest
 * branch            f8d7d77c06936315286eb55f8de22cd23c188571 -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/googletest': checked out 'f8d7d77c06936315286eb55f8de22cd23c188571'
From https://github.com/microsoft/GSL
 * branch            6f4529395c5b7c2d661812257cd6780c67e54afa -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/ms-gsl': checked out '6f4529395c5b7c2d661812257cd6780c67e54afa'
From https://github.com/nlohmann/json
 * branch            9cca280a4d0ccf0c08f47a99aa71d1b0e52f8d03 -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/nlohmann-json': checked out '9cca280a4d0ccf0c08f47a99aa71d1b0e52f8d03'
From https://github.com/open-telemetry/opentelemetry-proto
 * branch            2bd940b2b77c1ab57c27166af21384906da7bb2b -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/opentelemetry-proto': checked out '2bd940b2b77c1ab57c27166af21384906da7bb2b'
From https://github.com/opentracing/opentracing-cpp
 * branch            06b57f48ded1fa3bdd3d4346f6ef29e40e08eaf5 -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/opentracing-cpp': checked out '06b57f48ded1fa3bdd3d4346f6ef29e40e08eaf5'
From https://github.com/jupp0r/prometheus-cpp
 * branch            e5fada43131d251e9c4786b04263ce98b6767ba5 -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/prometheus-cpp': checked out 'e5fada43131d251e9c4786b04263ce98b6767ba5'
Submodule 'civetweb' (https://github.com/civetweb/civetweb.git) registered for path 'third_party/opentelemetry-cpp/third_party/prometheus-cpp/3rdparty/civetweb'
Submodule 'googletest' (https://github.com/google/googletest.git) registered for path 'third_party/opentelemetry-cpp/third_party/prometheus-cpp/3rdparty/googletest'
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/prometheus-cpp/3rdparty/civetweb'...
Cloning into '/home/builder/.termux-build/libgrpc/cache/tmp-checkout/third_party/opentelemetry-cpp/third_party/prometheus-cpp/3rdparty/googletest'...
From https://github.com/civetweb/civetweb
 * branch            d7ba35bbb649209c66e582d5a0244ba988a15159 -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/prometheus-cpp/3rdparty/civetweb': checked out 'd7ba35bbb649209c66e582d5a0244ba988a15159'
From https://github.com/google/googletest
 * branch            e2239ee6043f73722e7aa812a459f54a28552929 -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/third_party/prometheus-cpp/3rdparty/googletest': checked out 'e2239ee6043f73722e7aa812a459f54a28552929'
From https://github.com/Microsoft/vcpkg
 * branch              fba75d09065fcc76a25dcf386b1d00d33f5175af -> FETCH_HEAD
Submodule path 'third_party/opentelemetry-cpp/tools/vcpkg': checked out 'fba75d09065fcc76a25dcf386b1d00d33f5175af'
From https://github.com/protocolbuffers/protobuf
 * branch            74211c0dfc2777318ab53c2cd2c317a2ef9012de -> FETCH_HEAD
Submodule path 'third_party/protobuf': checked out '74211c0dfc2777318ab53c2cd2c317a2ef9012de'
From https://github.com/envoyproxy/protoc-gen-validate
 * branch            32c2415389a3538082507ae537e7edd9578c64ed -> FETCH_HEAD
Submodule path 'third_party/protoc-gen-validate': checked out '32c2415389a3538082507ae537e7edd9578c64ed'
From https://github.com/google/re2
 * branch            0c5616df9c0aaa44c9440d87422012423d91c7d1 -> FETCH_HEAD
Submodule path 'third_party/re2': checked out '0c5616df9c0aaa44c9440d87422012423d91c7d1'
From https://github.com/cncf/xds
 * branch            3a472e524827f72d1ad621c4983dd5af54c46776 -> FETCH_HEAD
Submodule path 'third_party/xds': checked out '3a472e524827f72d1ad621c4983dd5af54c46776'
From https://github.com/madler/zlib
 * branch            f1f503da85d52e56aae11557b4d79a42bcaa2b86 -> FETCH_HEAD
Submodule path 'third_party/zlib': checked out 'f1f503da85d52e56aae11557b4d79a42bcaa2b86'
~/.termux-build/libgrpc/cache
Downloading https://github.com/Kitware/CMake/releases/download/v4.0.3/cmake-4.0.3-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 52.9M  100 52.9M    0     0   129M      0 --:--:-- --:--:-- --:--:--  129M
Downloading https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  120k  100  120k    0     0   805k      0 --:--:-- --:--:-- --:--:--  805k
Archive:  /home/builder/.termux-build/libgrpc/tmp/ninja-1.12.1.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.12.1/ninja  
-- The C compiler identification is GNU 13.3.0
-- The CXX compiler identification is GNU 13.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test HAVE_STDC_FORMAT_MACROS
-- Performing Test HAVE_STDC_FORMAT_MACROS - Success
-- Performing Test ABSL_INTERNAL_AT_LEAST_CXX17
-- Performing Test ABSL_INTERNAL_AT_LEAST_CXX17 - Success
-- Performing Test ABSL_INTERNAL_AT_LEAST_CXX20
-- Performing Test ABSL_INTERNAL_AT_LEAST_CXX20 - Failed
CMake Error at third_party/cares/cares/CMakeLists.txt:1 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2025-07-24 00:35:12 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/16484410315">16484410315</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/libgrpc/build.sh
+++ b/packages/libgrpc/build.sh
@@ -3,7 +3,7 @@ TERMUX_PKG_DESCRIPTION="High performance, open source, general RPC framework tha
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_SRCURL=git+https://github.com/grpc/grpc
-TERMUX_PKG_VERSION="1.73.1"
+TERMUX_PKG_VERSION="1.74.0"
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_TAG_TYPE="latest-release-tag"
 TERMUX_PKG_DEPENDS="abseil-cpp, c-ares, ca-certificates, libc++, libprotobuf, libre2, openssl, protobuf, zlib"
@@ -41,7 +41,10 @@ termux_step_host_build() {
 	# when building version 1.17.2:
 	CXXFLAGS="-Wno-error=class-memaccess" \
 		CFLAGS="-Wno-implicit-fallthrough" \
-		cmake -G Ninja "$TERMUX_PKG_SRCDIR"
+		cmake \
+		-DCMAKE_POLICY_VERSION_MINIMUM=3.5 \
+		-G Ninja \
+		"$TERMUX_PKG_SRCDIR"
 
 	ninja grpc_cpp_plugin
 }

